python/foglamp/services/core/api/service.py

```diff
@@ -87,6 +89,10 @@ def get_service_records():
             raise web.HTTPBadRequest(reason='Missing type property in payload.')
         if not service_type in ['south', 'north']:
             raise web.HTTPBadRequest(reason='Only north and south types are supported.')
+        if enabled is not None:
+            if enabled not in ['t', 'f', 'true', 'false']:
+                raise web.HTTPBadRequest(reason='Only "t", "f", "true", "false" are allowed for value of enabled.')
+        enabled_changed = True if enabled.lower() in ['t', 'true'] else False
```

Comment:
is_enabled  is better name then it should be mapped to is_schedule_enabled when plugged with scheduler